File: c/1045.c
#include <stdio.h>

int main()
{

    double a, b, c, aux;
    scanf("%lf %lf %lf", &a, &b, &c);

    if (b > a && b > c)
    {
        aux = b;
        b = a;
        a = aux;
    }
    if (c > a && c > b)
    {
        aux = c;
        c = a;
        a = aux;
    }

    if (a >= b + c)
    {
        printf("NAO FORMA TRIANGULO\n");
        return 0;
    }
    if (a * a == (b * b + c * c))
    {
        printf("TRIANGULO RETANGULO\n");
    }
    if (a * a > (b * b + c * c))
    {
        printf("TRIANGULO OBTUSANGULO\n");
    }
    if (a * a < (b * b + c * c))
    {
        printf("TRIANGULO ACUTANGULO\n");
    }
    if (a == b && b == c)
    {
        printf("TRIANGULO EQUILATERO\n");
    }
    else if (a == b || a == c || b == c)
    {
        printf("TRIANGULO ISOSCELES\n");
    }
}

File: js/1045.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split(' ')
lines.sort()

let a = parseFloat(lines[0])
let b = parseFloat(lines[1])
let c = parseFloat(lines[2])


if (a >= b + c)
{
    console.log("NAO FORMA TRIANGULO");
    return 0;
}
if (a * a == (b * b + c * c))
{
    console.log("TRIANGULO RETANGULO");
}
if (a * a > (b * b + c * c))
{
    console.log("TRIANGULO OBTUSANGULO");
}
if (a * a < (b * b + c * c))
{
    console.log("TRIANGULO ACUTANGULO");
}
if (a == b && b == c)
{
    console.log("TRIANGULO EQUILATERO");
}
else if (a == b || a == c || b == c)
{
    console.log("TRIANGULO ISOSCELES");
}
